R/cohens_d.R

```diff
@@ -153,8 +153,8 @@ hedges_g <- function(x, y = NULL, data = NULL,
                      pooled_sd = TRUE, mu = 0, paired = FALSE,
                      ci = 0.95, alternative = "two.sided",
                      verbose = TRUE, ...) {
-  var.equal <- eval.parent(match.call()[["var.equal"]])
```

Comment:
Please change this back - we are matching arguments of `t.test()` which is `var.equal` not `var_equal`.